keyboards/edi/hardlight/mk1/keymaps/.gitignore

```diff
@@ -0,0 +1 @@
+fate
```

Comment:
This may be a better solution.
```sh
git update-index --skip-worktree --assume-unchanged keyboards/edg/hardlight/mk1/keymaps/fate
```

Reply:
My keymap contains leader key sequences with passwords. Hence, I don't want the keymap uploaded at all.